playbooks/ha_log_cfg_sync.yaml

```diff
@@ -0,0 +1,138 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+---
+- name: Config rsync daemon on log server slave node
+  hosts: logserver-slave, zuul-scheduler-slave
+  become: yes
+  tasks:
+    - file: path=/etc/rsyncd state=directory
+
+    - file: path=/var/run/rsyncd.pid state=absent
+
+    - name: Create rsyncd config file for logs and zuul config files sync
+      template:
+        src: "{{ labkeeper_config_git_dest }}/rsync/rsyncd.conf.j2"
+        dest: /etc/rsyncd/rsyncd.conf
+
+    - name: Start rsync daemon on zuul slave node
+      shell: |
+        echo 'root:openlab' > /etc/rsyncd/rsyncd.pass
+        chmod 600 /etc/rsyncd/rsyncd.pass
+        rsync --daemon --config=/etc/rsyncd/rsyncd.conf
+      args:
+        executable: /bin/bash
+
+- name: Config rsync client on zuul master node
+  hosts: logserver-master, zuul-scheduler-master
+  become: yes
+  tasks:
+    - file: path=/etc/rsync state=directory
+
+    - name: Create sync password file
+      copy:
+        dest: /etc/rsync/rsync_client.pwd
+        content: "openlab"
+        mode: 0600
+
+    - name: Create script for sync log files
+      copy:
+        dest: /etc/rsync/cron_sync_zuul_log.sh
+        content: |
+          #/bin/bash
+          rsync -auvrtzopgP --progress --password-file=/etc/rsync/rsync_client.pwd /srv/static/logs/ \
+          root@"{{ hostvars[groups['logserver-slave'][0]].ansible_host }}"::openlab_logserver
+        mode: 0755
+
+    - name: Add crontab task fo sync logs between master and slave
+      cron:
```

Comment:
The crontab is set in master node, what happen after master-slave switching?
We have to consider two way sync, because of master-slave switching

Reply:
the `master-slave switching` is not a real switch according to previous discussion, right ? after `master` down, `slave` become master, we will create a new slave.

Reply:
> the `master-slave switching` is not a real switch according to previous discussion, right ? after `master` down, `slave` become master, we will create a new slave.

When slave become new master, which script will config rsync crontab in new master host?